feature-libs/pickup-in-store/occ/adapters/occ-pickup-location.adapter.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { CartModification } from '@spartacus/cart/base/root';
+import { OccEndpointsService, PointOfService } from '@spartacus/core';
+import { PickupLocationAdapter } from '@spartacus/pickup-in-store/core';
+import {
+  SetPickupOptionToDeliveryPayload,
+  SetPickupOptionToPickupInStorePayload,
+} from '@spartacus/pickup-in-store/root';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class OccPickupLocationAdapter implements PickupLocationAdapter {
+  constructor(
+    protected http: HttpClient,
+    protected occEndpointsService: OccEndpointsService
+  ) {
+    // Intentional empty constructor
+  }
+
+  getStoreDetails(storeName: string): Observable<PointOfService> {
+    return this.http.get<PointOfService>(
+      this.occEndpointsService.buildUrl('storeDetails', {
+        urlParams: { storeName },
+        queryParams: { fields: 'FULL' },
+      })
+    );
+  }
+
+  setPickupOptionToDelivery(
```

Comment:
`setPickupOptionToDelivery` and `setPickupOptionToPickupInStore` can be removed. We have `update` in `OccCartEntryAdapter`.
```
public update(
    userId: string,
    cartId: string,
    entryNumber: string,
    qty?: number,
    pickupStore?: string,
    pickupToDelivery: boolean = false
  ): Observable<CartModification>
```
note: `pickupToDelivery: boolean`

Reply:
Done etPickupOptionToDelivery and setPickupOptionToPickupInStore is  removed and we are using the new cart methods